src/api/v1beta1/properties.go

```diff
@@ -186,21 +208,65 @@ func (dk *DynaKube) PullSecret() string {
 
 // ActiveGateImage returns the ActiveGate image to be used with the dk DynaKube instance.
 func (dk *DynaKube) ActiveGateImage() string {
-	return resolveImagePath(newActiveGateImagePath(dk))
+	if dk.CustomActiveGateImage() != "" {
+		return dk.CustomActiveGateImage()
+	}
+
+	apiUrlHost := dk.ApiUrlHost()
+
+	if apiUrlHost == "" {
+		return ""
+	}
+
+	return apiUrlHost + defaultActiveGateImage
+}
+
+func (dk *DynaKube) deprecatedActiveGateImage() string {
+	if dk.Spec.KubernetesMonitoring.Image != "" {
```

Comment:
if different images are set in `kubernetesMonitoring` and `routing`, the `routing` image would never be returned

Reply:
it is the same code as before but inlined. The problem is, one of the images must take precedent since we only have one image now. The kubernetes monitoring and routing parts are deprecated since v1alpha1, so it is information that is just lost in conversion